transmission_interface/include/transmission_interface/transmission_info.hpp

```diff
@@ -15,18 +15,47 @@
 #ifndef TRANSMISSION_INTERFACE__TRANSMISSION_INFO_HPP_
 #define TRANSMISSION_INTERFACE__TRANSMISSION_INFO_HPP_
 
+// C++ standard
+#include <vector>
 #include <string>
 
```

Comment:
```suggestion
```

Reply:
Vector is required, but the comment is removed